NewHaven/CMakeLists.txt

```diff
@@ -14,11 +14,11 @@ if(UNIX)
     endif(UNIX)
 include_directories(src)
 
-add_executable(VGMap test/VGMapDriver.cpp src/VGMap.cpp src/Resources.cpp)
-add_executable(GameBoardMapTest test/GBMapDriver.cpp src/GBMap.cpp src/Resources.cpp src/Square.cpp)
-add_executable(MapLoaderTest test/MapLoaderDriver.cpp src/VGMapLoader.cpp src/VGMap.cpp src/GBMapLoader.cpp src/GBMap.cpp src/Square.cpp Exceptions/InvalidConfigurationException.cpp Exceptions/BoardConfigurationNotLoaded.cpp src/Resources.cpp)
+add_executable(VGMap src/VGMap.cpp src/Resources.cpp)
+add_executable(GameBoardMapTest src/GBMap.cpp src/Resources.cpp src/Square.cpp)
+add_executable(MapLoaderTest src/VGMapLoader.cpp src/VGMap.cpp src/GBMapLoader.cpp src/GBMap.cpp src/Square.cpp Exceptions/InvalidConfigurationException.cpp Exceptions/BoardConfigurationNotLoaded.cpp src/Resources.cpp)
 add_executable(ResourcesTest test/ResourcesDriver.cpp src/Resources.cpp src/ResourceTracker.cpp)
-add_executable(ScoreTest test/ScoreDriver.cpp src/GBMap.cpp src/Resources.cpp src/Square.cpp src/VGMap.cpp src/ScoreCalculator.cpp src/ResourceTracker.cpp)
+add_executable(ScoreTest src/GBMap.cpp src/Resources.cpp src/Square.cpp src/VGMap.cpp src/ScoreCalculator.cpp src/ResourceTracker.cpp)
```

Comment:
Why keep executables without `main()`? Weren't we going to remove all the driver that aren't relevant to asg2?